packages/peer-exchange/src/waku_peer_exchange.ts

```diff
@@ -1,33 +1,28 @@
 import type { Stream } from "@libp2p/interface-connection";
-import { ConnectionManager } from "@libp2p/interface-connection-manager";
 import { PeerId } from "@libp2p/interface-peer-id";
 import type { Peer, PeerStore } from "@libp2p/interface-peer-store";
+import {
+  getPeersForProtocol,
+  selectConnection,
+  selectPeerForProtocol,
+} from "@waku/core";
 import { ENR } from "@waku/enr";
 import {
   PeerExchange,
+  PeerExchangeComponents,
   PeerExchangeQueryParams,
   PeerExchangeResponse,
-  PeerInfo,
   ProtocolOptions,
 } from "@waku/interfaces";
 import all from "it-all";
-// import all from "it-all";
 import * as lp from "it-length-prefixed";
 import { pipe } from "it-pipe";
 import { Uint8ArrayList } from "uint8arraylist";
 
-import { selectConnection } from "../select_connection";
-import { getPeersForProtocol, selectPeerForProtocol } from "../select_peer";
-
-import { PeerExchangeRPC } from "./peer_exchange_rpc";
+import { PeerExchangeRPC } from "./rpc";
```

Comment:
```suggestion
import { PeerExchangeRPC } from "./rpc.js";
```

tech debt: `@waku/core` has a script `./build-scripts/fix-import.js` that automatically adds all the `.js` extensions to imports for ESM.

I am not convinced this was the best way forward so I did not setup this scripts for all other packages.

The issue now is that the missing `.js` can be unspotted until we release and use it in a different project.

Tracked here https://github.com/waku-org/js-waku/issues/1039